.travis.yml

```diff
@@ -16,7 +16,7 @@ deploy:
       secure: ahwA9BC1ZWka/XABCDmaaetk1inNE6FnjndDL8PWQUEUFdai37pAr31n0/y0lSAcghnBrRQkekyHOZ0ddqBSoI0DRtqrP2B2nkZo/4eC/7q7y2QRl53yb/tvjvsI14fn5aSkyfdywMU5qOjOHpPDF5nk1scNpIDu3QlAuTWlN6o=
     on:
       tags: true
-      node: '0.10'
+      node: '8'
```

Comment:
8 is the current LTS release of Node, and figured that since it's Node's "recommended" release on their site, it seems like the one that we'd want to make sure it's working for before deploying.

Reply:
Ah, that makes sense. :+1: